### DISCUSSION TITLE
ToolDiy/software/texmacs/

### DISCUSSION THREAD

**Discussion by giscus** [cargo-youth/ToolDiy#4] [Announcements]
# ToolDiy/editor/texmacs/

https://cargo-youth.github.io/ToolDiy/editor/texmacs/

**Comment by Flurry-L** [comment]
TeXmacs 编辑器可以试试[墨干编辑器](https://github.com/XmacsLabs/mogan)。

**Comment by jackyliu16** [comment]
有個視頻我感覺可以看一下
【太丝滑了！数学/科研 论文神器-0学习成本-TeXmacs/墨干入门教程-如何自定义快捷键-哔哩哔哩】 https://b23.tv/pjFSASG
